onebusaway-android/src/main/java/org/onebusaway/android/ui/ArrivalsListFragment.java

```diff
@@ -1638,6 +1639,23 @@ public boolean equals(Object obj) {
         }
     }
 
+    private void reportAnalytics(ObaSituation situation) {
+        ObaSituation.AllAffects[] allAffects = situation.getAllAffects();
+        List<String> agencyIds = new ArrayList<>();
```

Comment:
Could you change `agencyIds` to a `HashSet` instead of a `List`?  I doubt we'll ever have enough agencies in a region for iterating through the list in the `.contains()` method to become problematic performance-wise, but just to be sure... :)